libs/model/src/middleware/authorization.ts

```diff
@@ -149,6 +149,8 @@ async function buildAuth(
       address: actor.address,
       community_id: auth.community_id,
       role: { [Op.in]: roles },
+      verified: { [Op.ne]: null },
+      // TODO: check verification token expiration
```

Comment:
still todo?

Reply:
yes, this is linked to the API auth work @timolegros ?